FUTURE-NATIVE.md

```diff
@@ -0,0 +1,29 @@
+Lots of people have asked about reason-react compiled to native (Reason Native React Native or RNRN 😎), here's a rough outline of dependencies that have to fall into place.
+
+# Prerequisites
+status: in progress
+
+The main thing here is getting a package manager + build toolchain that make cross-compilation a breeze. @jordwalke and some others are working on this right now, and it's called [esy](https://github.com/esy/esy).
+
+Time remaining before it's usable: 2-8 months? :P [TODO put in a better estimate about timeline]
```

Comment:
Here's a PR you can follow for cross-target compilation progress: https://github.com/esy/esy/pull/169.
It seamlessly compiles any basic dune projects but doesn't work for older ocamlbuild setups without busting `OCAML_TOPLEVEL_PATH` (due to topfind constraints).